README.md

```diff
@@ -32,53 +45,102 @@ This software should :
 
 *Tick the tasks once successfully implemented and tested*
 
-It is assumed that the sensors' data is not processed in any way by the embedded computer. The data received is raw and should be converted to readable values by the ground station.
-
->NB : how do we deal with the calibration of the sensors ?
-
 
 # Hardware
 
-The ground station is made of different systems :
+The hardware is made of the following elements :
+  * Rocket
   * The Launch Pad Station
   * The Telemetry Receiver 
-  * The Dashboard Computer
+  * The Ground Station
   * The FPV System (*not described here*)
 
 
+Here is a diagram of the data links between the subsystems :
+
+![data_link](/doc/diagrams/data_links.png)
+
+
+>The diagram was made with [draw.io](https://www.draw.io)<br>
+>To make changes to it, edit the source file `/doc/diagrams/data_links.xml`
+
+
+The Telemetry Interface and Launch Pad Station Interface are only forwarders. There is no logic embeded into them.
+
+Here is a schematic of the data flow trough the interfaces :
 
+![telemetry_link](/doc/diagrams/interface_link.png)
 
-## Launch Pad Station
+`BONJOUR` is a unique string sent on serial connection initiation that is used to identify the Telemetry Interface and the LPS Interface amoung all the serial devices connected to the Dashboard Computer
 
-## Telemetry Receiver
 
-## Dashboard Computer
+# Sensors
 
-This software should run on a standard computer running Windows. It should work as well on Unix systems but has not been tested.
+The following sensors are embedded on the Rocket :
+  * Inertial Motion Unit `ICM20602` from Invensense
+  * Static pressure sensor `BMP280` from Bosch
+  * Magnetometer `LIS3MDLTR` from STMicroelectronics
+  * Dynamic pressure sensor `ABPDRRT005PG2A5` from Honeywell
+  * GPS receiver `M8Q` from u-blox
 
-All the received data is saved in the Dashboard Computer hard drive for future use.
 
 # Transmission protocol
 
-See the example telemetry code [here](./dummy_telemetry/dummy_telemetry.ino)
+For phase 1, the above schematic should be self explaining.
 
-# Installation
+For phase 2, see [this issue](https://github.com/aesirkth/GroundStation/issues/4)
 
-Install `python 3.7.4`
 
-Run `python3 -m pip install -r requirements.txt`
+# File storage
+
+See [this issue](https://github.com/aesirkth/GroundStation/issues/4)
+
+
+# How to use ?

```

Comment:
Under this comment we could just shortly write something like "Connect the TM Rx into a USB-port, run the program(with or without instruction on how to do that), start the Tx, preferably through the rocket STM."

Reply:
Yes, but we don't really have information on the process needed to get the STM running. I was planning to put this description in the section "Flight conditions" line 127 and just focus on the test Arduino board for now ("Testing" section)